org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/json/schema/Tuple.java

```diff
@@ -0,0 +1,29 @@
+package org.eclipse.wildwebdeveloper.json.schema;
+
+public class Tuple<T, U> {
```

Comment:
You may prefer to import org.eclipse.xtext.xbase.lib (that's already a transitive dependency) and use its Pair class.